### PR TITLE
Search results component

### PR DESCRIPTION
**Pull request by yordanalexandrov**
Only for review, will not be merged in this state.

### CODE REVIEW

File: projects/onto-search/src/lib/onto-search-results/onto-search-results.component.scss [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+
+
+
+
+//.onto-table thead, .onto-table tbody, .onto-table tfoot,
+//[onto-table-header-row], [onto-table-row], [onto-table-footer-row] {
+//  background: inherit;
+//}
```
Comment (svilenvelikov) [inline projects/onto-search/src/lib/onto-search-results/onto-search-results.component.scss]: What is this sorcery?

Comment (yordanalexandrov) [reply projects/onto-search/src/lib/onto-search-results/onto-search-results.component.scss]: ¯\\_(ツ)_/¯

File: projects/onto-search/tslint.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../../tslint.json",
+//  "extends": "../../tslint.json",
```
Comment (svilenvelikov) [inline projects/onto-search/tslint.json]: Why this is commented out?

File: projects/onto-search/src/lib/onto-search-results/onto-search-results.component.html [hunk 3]
```diff
@@ -8,7 +8,13 @@
       </th>
       <td cdk-cell class="onto-table-cell {{'onto-table-column-' + columnConfig.name}}" *cdkCellDef="let row"
           appCypressData="onto-table-cell">
-        <div>{{columnConfig.dataFunction(row)}}</div>
+        <div>
+          <ng-container *ngIf="columnConfig.dataFunction; else cellTemplate">
+            {{columnConfig.dataFunction(row)}}
+          </ng-container>
+          <ng-template #cellTemplate [ngTemplateOutlet]="columnConfig.dataTemplate"
```
Comment (svilenvelikov) [inline projects/onto-search/src/lib/onto-search-results/onto-search-results.component.html:15]: good :)

File: projects/onto-search/src/lib/onto-search-results/onto-search-results.component.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  @Input()
+  public config: TableConfig;
+
+  @Input()
+  componentClass: string | string[];
+
+  displayColumns: string[];
+
+
+  constructor() {
```
Comment (svilenvelikov) [inline projects/onto-search/src/lib/onto-search-results/onto-search-results.component.ts]: removethis empty constructor

**Review by svilenvelikov — APPROVED** [review]

File: cypress/steps/search-results-steps.ts [hunk 5]
```diff
@@ -0,0 +1,37 @@
+export default class SearchResultsSteps {
+  static visit(waitLoader: boolean = true) {
```
Comment (svilenvelikov) [inline cypress/steps/search-results-steps.ts]: remove this `waitLoader: boolean = true`

Comment (svilenvelikov) [reply cypress/steps/search-results-steps.ts]: I know there is similar shit in the UI app too, but I'll remove it. :)

File: cypress/test/search-results.spec.ts [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import SearchResultsSteps from '../steps/search-results-steps';
+
+describe('Onto Search Results Component', () => {
+  it('Should find the component', () => {
+    SearchResultsSteps.visit();
+    SearchResultsSteps.getOntoSearchResultsComponent();
```
Comment (svilenvelikov) [inline cypress/test/search-results.spec.ts]: I know that cypress will blow if it's not there, but better add `.should('be.visible')` at least

File: cypress/test/search-results.spec.ts [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+  });
+
+  it('Should have data loaded', () => {
+    SearchResultsSteps.visit();
+
+    SearchResultsSteps.getDataRows();
```
Comment (svilenvelikov) [inline cypress/test/search-results.spec.ts]: Maybe verify how many rows there are in the table?

**Review by teodossidossev — APPROVED** [review]
lgtm

**Pull request merged**